compose/ui/ui-test-junit4/src/skikoTest/kotlin/androidx/compose/ui/test/ComposeUiSkikoTestTest.kt

```diff
@@ -368,6 +369,7 @@ class ComposeUiSkikoTestTest {
     }
 
     @Test
+    @Ignore // TODO(ivan.matkov): Bug in SkikoInputDispatcher - it uses overload for mouse input
```

Comment:
We should remove it after merging https://github.com/JetBrains/compose-multiplatform-core/pull/688/files